x-pack/plugins/cases/server/saved_object_types/cases_oracle.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectsType } from '@kbn/core/server';
+import { ALERTING_CASES_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
+import { schema } from '@kbn/config-schema';
+import { CASE_ORACLE_SAVED_OBJECT } from '../../common/constants';
+
+export const casesOracleSavedObjectType: SavedObjectsType = {
+  name: CASE_ORACLE_SAVED_OBJECT,
+  indexPattern: ALERTING_CASES_SAVED_OBJECT_INDEX,
+  hidden: true,
+  /**
+   * TODO: Verify
+   */
+  namespaceType: 'agnostic',
+  mappings: {
+    dynamic: false,
+    properties: {
+      cases: {
+        properties: {
+          id: {
+            type: 'keyword',
+          },
+        },
+      },
+      counter: {
+        type: 'unsigned_long',
+      },
+      createdAt: {
+        type: 'date',
+      },
+      /*
+      grouping: {
```

Comment:
Should this still be commented out?

Reply:
No, we do not want to index the grouping at the moment. No plans to query the oracle about grouping. The other data are needed for telemetry purposes. They are not actually needed by the connector. It may change as we develop though.